lib/calendar/bloc.dart

```diff
@@ -12,36 +12,19 @@ import 'data.dart';
 class CalendarBloc {
   const CalendarBloc();
 
-  CacheController<List<Event>> fetchEvents() {
-    final storage = services.storage;
-    storage.cache.clear();
-    return fetchList(
-      makeNetworkCall: () => services.network.get(
-        'calendar',
-        parameters: {
-          // We have to set this query parameter because otherwise—you guessed
-          // it—no events are being returned at all 😂
-          'all': 'true',
-        },
-      ),
-      parser: (data) => Event.fromJson(data),
-      serviceIsPaginated: false,
-    );
-  }
-
   Stream<CacheUpdate<List<Event>>> fetchTodaysEvents() {
     // The great, thoughtfully designed Calendar API presents us with daily
     // challenges, such as: How do I get today's events?
     // And the simple but ingenious answer to that is:
-    // 1. Download all events (every time). (≈50 kb using the demo account)
+    // 1. Download all events (every time). (≈ 50 kb using the demo account)
     // 2. Implement your own logic to filter them. Have fun 😊
-    final allEvents = fetchEvents()..fetch();
+    final allEvents = services.storage.root.events.controller;
     return allEvents.updates.map((u) {
       if (!u.hasData) {
         return u;
       }
 
-      return CacheUpdate(
+      return CacheUpdate.raw(
```

Comment:
I think we can now use `CacheController.map` instead of directly working on a stream of `CacheUpdate`s